### PR TITLE
Nh ramp pid

### PR DESCRIPTION
**Pull request by noahheller**
(not done)

### CODE REVIEW

**Review by batchen1 — DISMISSED** [review]

File: src/main/java/frc/robot/subsystems/Ramp.java [hunk 1]
```diff
@@ -34,6 +23,11 @@ public Ramp() {
     private void configureMotor() {
         neoPidMotor.setSmartMotionAllowedClosedLoopError(Constants.RAMP_ERROR_RANGE);
         neoPidMotor.setMaxAccel(Constants.RAMP_MAX_RPM_ACCELERATION);
+        neoPidMotor.setSmartMotionAllowedClosedLoopError(Constants.RAMP_ERROR_RANGE);
```
Comment (batchen1) [inline src/main/java/frc/robot/subsystems/Ramp.java]: you have 2 duplicate lines

File: src/main/java/frc/robot/subsystems/Ramp.java [hunk 2]
```diff
@@ -34,6 +23,11 @@ public Ramp() {
...
+        neoPidMotor.setMaxAccel(Constants.RAMP_MAX_RPM_ACCELERATION);
+        neoPidMotor.getPidController().setP(Constants.RAMP_PID_P);
+        neoPidMotor.getPidController().setFF(Constants.RAMP_PID_FAR_FF);
+        resetEncoder();
```
Comment (batchen1) [inline src/main/java/frc/robot/subsystems/Ramp.java]: this one is duplicate too

**Review by batchen1 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Ramp.java (hunk 1 above)
Comment (noahheller) [reply src/main/java/frc/robot/subsystems/Ramp.java]: done

File: src/main/java/frc/robot/subsystems/Ramp.java (hunk 2 above)
Comment (noahheller) [reply src/main/java/frc/robot/subsystems/Ramp.java]: done

**Review by batchen1 — APPROVED** [review]

**Pull request merged**